ui/plugin-system/src/components/PluginLoadingBoundary/index.ts

```diff
@@ -0,0 +1,14 @@
+// Copyright 2021 The Perses Authors
```

Comment:
mmm that's weird the syntax highlight of github doesn't work anymore here :/

Reply:
Haha, same for me. PR is too big. 😂